src/components/TextInput/index.jsx

```diff
@@ -1,7 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
+import styles from "./styles/styles.module.scss";
 
-const TextInput = () => {
-  return <p>text input</p>;
+const TextInput = ({ title, minlength }) => {
+  let [inputStyle, setInputStyle] = useState(styles.input);
+  let [errorText, setErrorText] = useState("");
+
+  const handleInputValue = (event) => {
+    if (event.target.value.length === 0) {
+      setInputStyle(`${inputStyle} ${styles.error} ${styles.errorBox}`);
+      setErrorText("can't be empty");
+    } else if (event.target.value.length < 2) {
+      setInputStyle(`${inputStyle} ${styles.error} ${styles.errorBox}`);
+      setErrorText("must be at least 2 characters long");
+    }
+  };
+
+  return (
+    <label className={styles.label}>
+      {title}
+      <input
+        className={inputStyle}
+        type="text"
+        minlength={minlength}
+        required
+        onBlur={(event) => {
+          handleInputValue(event);
+        }}
```

Comment:
Check if this works:
```suggestion
        onBlur={handleInputValue}
```

Reply:
yes it works thanks